seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/sql/zeta/ZetaSQLFunction.java

```diff
@@ -665,4 +699,180 @@ private Object executeBinaryExpr(BinaryExpression binaryExpression, Object[] inp
+            SeaTunnelRowType outRowType,
+            boolean isUsingOuter,
+            String alias) {
+        ExpressionList<?> expressions = lateralViewFunction.getParameters();
+        int aliasFieldIndex = outRowType.indexOf(alias);
+        for (Expression expression : expressions) {
+            if (expression instanceof Column) {
+                String column = ((Column) expression).getColumnName();
+                List<SeaTunnelRow> next = new ArrayList<>();
+                for (SeaTunnelRow row : seaTunnelRows) {
+                    int fieldIndex = outRowType.indexOf(column);
+                    Object splitFieldValue = row.getField(fieldIndex);
+                    transformExplodeValue(
+                            splitFieldValue,
+                            outRowType,
+                            isUsingOuter,
+                            next,
+                            aliasFieldIndex,
+                            row,
+                            expression,
+                            true);
+                }
+                seaTunnelRows = next;
+            } else if (expression instanceof Function) {
+                List<SeaTunnelRow> next = new ArrayList<>();
+                for (SeaTunnelRow row : seaTunnelRows) {
+                    Object splitFieldValue = computeForValue(expression, row.getFields());
+                    transformExplodeValue(
+                            splitFieldValue,
+                            outRowType,
+                            isUsingOuter,
+                            next,
+                            aliasFieldIndex,
+                            row,
+                            expression,
+                            false);
+                }
+                seaTunnelRows = next;
+            }
+        }
+        return seaTunnelRows;
+    }
+
+    private void transformExplodeValue(
+            Object splitFieldValue,
+            SeaTunnelRowType outRowType,
+            boolean isUsingOuter,
+            List<SeaTunnelRow> next,
+            int aliasFieldIndex,
+            SeaTunnelRow row,
+            Expression expression,
+            boolean keepValueType) {
+        if (splitFieldValue == null) {
+            if (isUsingOuter) {
+                next.add(copySeaTunnelRow(outRowType.getTotalFields(), row, aliasFieldIndex, null));
+            }
+            return;
+        }
+        if (splitFieldValue.getClass().isArray()) {
+            if (ArrayUtils.isEmpty((Object[]) splitFieldValue)) {
+                if (isUsingOuter) {
+                    next.add(
+                            copySeaTunnelRow(
+                                    outRowType.getTotalFields(), row, aliasFieldIndex, null));
+                }
+                return;
+            }
+            for (Object fieldValue : (Object[]) splitFieldValue) {
+                Object value = keepValueType ? fieldValue : String.valueOf(fieldValue);
```

Comment:
if the fieldValue is null, it will be convert to `"null"` by `String.valueOf(fieldValue)`. The value will be wrong.